appengine/Dockerfile

```diff
@@ -54,7 +55,7 @@ ENV PATH /rbenv/shims:/rbenv/bin:$PATH
 
 # Preinstall ruby runtimes.
 # The LAST version in the list is set as the default.
-ENV PREINSTALLED_RUBY_VERSIONS 2.0.0-p647 2.1.7 2.2.3
+ENV PREINSTALLED_RUBY_VERSIONS 2.0.0-p648 2.1.8 2.2.3 2.2.4
```

Comment:
Using the GitHub releases feature would be useful. We tried upgrading to ruby 2.2.4 as we saw this has been merged but the deploys are failing. The gcr.io image must not be pushed yet?